--- a/bin/init.d/shinken     2011-09-22 20:14:20.470007468 +0000
+++ b/bin/init.d/shinken     2011-09-22 20:24:58.742020660 +0000
@@ -353,7 +353,9 @@
     statusoutput=$(do_status "$1")
     rc=$?
     if [ $rc -ne 0 ]; then
-        failuremsg="Couldn't get status of $1 : $statusoutput"
+        failuremsg="Couldn't get status of $1 : Assuming already stopped"
+        log_success_msg "$failuremsg"
+        rc=0
     else
         stopoutput=$(do_stop "$1" 2>&1)
         rc=$?
