This patch fixes manpages in shinken

diff --git a/man/shinken-arbiter.8shinken b/man/shinken-arbiter.8shinken
index e74e746..d36b645 100644
--- a/man/shinken-arbiter.8shinken
+++ b/man/shinken-arbiter.8shinken
@@ -1,6 +1,20 @@
-=head1 SHINKEN-ARBITER                                                                   
-
-Command - Shinken arbiter command.
+.ie \nF \{\
+.    de IX
+.    tm Index:\\$1\t\\n%\t"\\$2"
+..
+.    nr % 0
+.    rr F
+.\}
+.el \{\
+.    de IX
+..
+.\}
+.\"
+.IX Title "shinken-arbiter"
+.TH shinken
+.SH NAME
+
+shinken-arbiter - Shinken arbiter command.
 
 =head1 SYNOPSIS
 
diff --git a/man/shinken-broker.8shinken b/man/shinken-broker.8shinken
index 7cf1951..3db800a 100644
--- a/man/shinken-broker.8shinken
+++ b/man/shinken-broker.8shinken
@@ -1,6 +1,20 @@
-=head1 SHINKEN-BROKER
-
-Command - Shinken broker command.
+.ie \nF \{\
+.    de IX
+.    tm Index:\\$1\t\\n%\t"\\$2"
+..
+.    nr % 0
+.    rr F
+.\}
+.el \{\
+.    de IX
+..
+.\}
+.\"
+.IX Title "shinken-broker"
+.TH shinken
+.SH NAME
+
+shinken-broker - Shinken broker command.
 
 =head1 SYNOPSIS
 
diff --git a/man/shinken-discovery.8shinken b/man/shinken-discovery.8shinken
index 34f0c39..4ba7cd1 100644
--- a/man/shinken-discovery.8shinken
+++ b/man/shinken-discovery.8shinken
@@ -1,6 +1,20 @@
-=head1 SHINKEN-DISCOVERY                                                                   
-
-Command - Shinken discovery command.
+.ie \nF \{\
+.    de IX
+.    tm Index:\\$1\t\\n%\t"\\$2"
+..
+.    nr % 0
+.    rr F
+.\}
+.el \{\
+.    de IX
+..
+.\}
+.\"
+.IX Title "shinken-discovery"
+.TH shinken
+.SH NAME
+
+shinken-discovery - Shinken discovery command.
 
 =head1 SYNOPSIS
 
diff --git a/man/shinken-poller.8shinken b/man/shinken-poller.8shinken
index 0de285a..295778b 100644
--- a/man/shinken-poller.8shinken
+++ b/man/shinken-poller.8shinken
@@ -1,6 +1,20 @@
-=head1 SHINKEN-POLLER                                                                   
-
-Command - Shinken poller command.
+.ie \nF \{\
+.    de IX
+.    tm Index:\\$1\t\\n%\t"\\$2"
+..
+.    nr % 0
+.    rr F
+.\}
+.el \{\
+.    de IX
+..
+.\}
+.\"
+.IX Title "shinken-poller"
+.TH shinken
+.SH NAME
+
+shinken-poller - Shinken poller command.
 
 =head1 SYNOPSIS
 
diff --git a/man/shinken-reactionner.8shinken b/man/shinken-reactionner.8shinken
index f0ddfde..3b68f68 100644
--- a/man/shinken-reactionner.8shinken
+++ b/man/shinken-reactionner.8shinken
@@ -1,6 +1,20 @@
-=head1 SHINKEN-REACTIONNER
-
-Command - Shinken reactionner command.
+.ie \nF \{\
+.    de IX
+.    tm Index:\\$1\t\\n%\t"\\$2"
+..
+.    nr % 0
+.    rr F
+.\}
+.el \{\
+.    de IX
+..
+.\}
+.\"
+.IX Title "shinken-reactionner"
+.TH shinken
+.SH NAME
+
+shinken-reactionner - Shinken reactionner command.
 
 =head1 SYNOPSIS
 
diff --git a/man/shinken-receiver.8shinken b/man/shinken-receiver.8shinken
index b4d08fc..91793ca 100644
--- a/man/shinken-receiver.8shinken
+++ b/man/shinken-receiver.8shinken
@@ -1,6 +1,20 @@
-=head1 SHINKEN-RECEIVER                                                                   
-
-Command - Shinken receiver command.
+.ie \nF \{\
+.    de IX
+.    tm Index:\\$1\t\\n%\t"\\$2"
+..
+.    nr % 0
+.    rr F
+.\}
+.el \{\
+.    de IX
+..
+.\}
+.\"
+.IX Title "shinken-receiver"
+.TH shinken
+.SH NAME
+
+shinken-receiver - Shinken receiver command.
 
 =head1 SYNOPSIS
 
diff --git a/man/shinken-scheduler.8shinken b/man/shinken-scheduler.8shinken
index 2b4e12f..f35dc54 100644
--- a/man/shinken-scheduler.8shinken
+++ b/man/shinken-scheduler.8shinken
@@ -1,6 +1,20 @@
-=head1 SHINKEN-SCHEDULER
-
-Command - Shinken scheduler command.
+.ie \nF \{\
+.    de IX
+.    tm Index:\\$1\t\\n%\t"\\$2"
+..
+.    nr % 0
+.    rr F
+.\}
+.el \{\
+.    de IX
+..
+.\}
+.\"
+.IX Title "shinken-scheduler"
+.TH shinken
+.SH NAME
+
+shinken-scheduler - Shinken scheduler command.
 
 =head1 SYNOPSIS
 
